README.md

```diff
@@ -27,19 +27,40 @@ We encourage others to contribute configurations they find useful. There is an [
 
 `%%gremlin` - Executes a Gremlin query against your database using web sockets. The results are similar to what the Gremlin console would return.
 
+`%%graph_notebook_config` - Sets the executing notebook's configuration to the JSON payload provided in the cell body.
+
+`%%graph_notebook_vis_options` - Sets the executing notebook's vis.js options to the JSON payload provided in the cell body.
+
+`%%neptune_ml` - Set of commands to integrate with NeptuneML functionality. [Documentation](https://aws.amazon.com/neptune/machine-learning/)
+
+
 **TIP** :point_right:  There is syntax highlighting for both `%%sparql` and `%%gremlin` queries to help you structure your queries more easily.
 
 #### Notebook line 'magic' extensions in the IPython 3 kernel
-`%graph_notebook_config` - Returns a JSON object that contains connection information for your host.
+`%gremlin_status` - Obtain the status of Gremlin queries. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/gremlin-api-status.html)
+
+`%sparql_status` - Obtain the status of SPARQL queries. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/sparql-api-status.html)
+
+`%load` - Generate a form to submit a bulk loader job. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/bulk-load.html)
 
-`%query_mode` - Lets you set the query mode for your queries to one of:
+`%load_ids` - Get ids of bulk load jobs. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/load-api-reference-status-examples.html)
 
-* `query` (the default) : executes the query against the normal SPARQL or Gremlin endpoint
-* `explain` : Returns an explanation of the query plan instead of the query's results (valid for both SPARQL and Gremlin).
-* `profile` : Returns a profile of the query's operation, but does not actually execute the query (valid only for Gremlin).
+`%load_status` - Get the status of a provided `load_id`. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/load-api-reference-status-examples.html)
+
+`%neptune_ml` - Set of commands to integrate with NeptuneML functionality. [Documentation](https://aws.amazon.com/neptune/machine-learning/)
+
+`%status` - Check the Health Status of the configured host endpoint. [Documentation](https://docs.aws.amazon.com/neptune/latest/userguide/access-graph-status.html)
 
 `%seed` - Provides a form to add data to your graph without the use of a bulk loader. both SPARQL and Gremlin have an airport routes dataset.
 
+`%graph_notebook_config` - Returns a JSON object that contains connection information for your host.
```

Comment:
It's a nitpick, but can we match the wording for the %% version and the % version? JSON object to "JSON payload", and instead of "connection information for your host" use "configuration information for connecting to your database"?